src/actions/collectiblesActions.js

```diff
@@ -229,3 +236,43 @@ export const fetchAllCollectiblesDataAction = (forAllAccounts?: boolean) => {
     }
   };
 };
+
+export const updateCollectibleTransactionAction = (hash: string) => {
+  return async (dispatch: Dispatch, getState: GetState, api: SDKWrapper) => {
+    const {
+      session: { data: { isOnline } },
+      collectibles: { transactionHistory: collectiblesHistory },
+    } = getState();
+    if (!isOnline) return;
+
+    const trxInfo = await getTrxInfo(api, hash);
+    if (!trxInfo) return;
+    const {
+      txInfo,
+      txReceipt,
+      nbConfirmations,
+      status,
+    } = trxInfo;
+
+    const accounts = Object.keys(collectiblesHistory);
```

Comment:
I think you can re-use `updateHistoryRecord` helper here

Reply:
Was willing to do so, but collectibles' transactions and other transactions differ in props, hence `updateHistoryRecord` outputs turn out to be different. I could alter typing for this method, but then it goes on an on in other usage places. 

Didn't want to align collectibles and other transactions too much as I would just be adding empty unnecessary data. So I've gone with this approach.

Reply:
Gotcha! While you're using this in one place only - I agree, we could keep that approach and refactor once we need to use it somewhere else